pubspec.lock

```diff
@@ -587,5 +579,5 @@ packages:
     source: hosted
     version: "3.1.1"
 sdks:
-  dart: ">=2.19.0 <3.0.0"
+  dart: ">=3.0.0-0 <4.0.0"
```

Comment:
Hmm should this be considered a breaking change?

Reply:
This is weird. Just checked on pub.dev that none of the updated transitive dependencies require dart 3.0, plus we have `sdk: ">=2.12.0 <3.0.0"` in `pubspec.yaml`. 🤔

Latest flutter stable does require 3.0, I wonder if by running `flutter pub` commands it updates the lockfile?

Reply:
I think this is due to building locally using the latest flutter. Does the latest flutter require dart 3? Also this is just a lock file so no breaking change here

Reply:
Turns out Dart recommends not committing lock files for libraries, so I removed it (merged into this branch)

https://github.com/RevenueCat/purchases-flutter/pull/701